lib/active_record/connection_adapters/cockroachdb/schema_statements.rb

```diff
@@ -57,10 +57,34 @@ def type_to_sql(type, limit: nil, precision: nil, scale: nil, array: nil, **) #
             end
           # The call to super might have appeneded [] already.
           if array && type != :primary_key && !sql.end_with?("[]")
-            sql = "#{sql}[]" 
+            sql = "#{sql}[]"
           end
           sql
         end
+
+        # Resets the sequence of a table's primary key to the maximum value.
```

Comment:
Can you update the comment to say that this overrides the method from the Postgres adapter?